Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -271,6 +273,7 @@ extension PostReceiptDataOperation.PostData: Encodable {
         case paywall
         case testReceiptIdentifier = "test_receipt_identifier"
         case appTransaction = "app_transaction"
+        case metadata = "metadata"
```

Comment:
Hmm nitpick but is it needed to specify the key?
```suggestion
        case metadata
```